### ISSUE TITLE
All browsers: NVDA jumps to the very left cell coresponding to the first cell of merged cells in a table when using table navigation commands

### ISSUE THREAD

**Issue by Adriani90** [nvaccess/nvda#9262]
### Steps to reproduce:
1. Open NVDA
2. Open this article or any other example containing a table and some merged cells in a row.
https://www.capitaliq.com/CIQDotNet/CreditResearch/RenderArticle.aspx?articleId=2164661&SctArtId=466758&from=CM&nsl_code=LIME&sourceObjectId=10864796&sourceRevId=1&fee_ind=N&exp_date=20290210-18:20:17&mkt_tok=eyJpIjoiWkdKa01XRm1ZMlU1WW1KbSIsInQiOiIwcGJmREFPUUdHYlQ2VjJMSURvbDQ4cmY3Z0txTk5VR1JJOE43TVNGUmt3cUg3MVwvbE4xRUpoYVhRXC95WnJTXC8wNXhPQ1g2eWZGT2NIVk1BSkM1MTFRWk9VbjdubEQrOGE4UHhOTlJCb2M5bmZIcW13ZGRMSUlTNUVZdFdUQkY5ZiJ9
3. While in table, use ctrl+alt+arrow keys to navigate through the table (in this example is table 2)
4. In this example, navigate to "china" and then go to the next row
5. Navigate back to the first row where countries are ordered.

### Actual behavior:
The secont row contains merged cells. When navigating from China to the second row, NVDA moves the virtual cursor to the first cell at the left border of merged cells. When navigating back to the first row, the cursor is then on Argentina.

### Expected behavior:
NVDA should retain the position on rows which do not have merged cells. In this example, when pressing ctrl+alt+up arrow, The virtual cursor should be on China because this was the country where I started my navigation downwards from.

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
2017.4

#### Windows version:
Windows 8.1

#### Name and version of other software in use when reproducing the issue:
Firefox 65, IE11, Chrome 71.

#### Does the issue still occur after restarting your PC?
yes

#### Have you tried any other versions of NVDA?
not yet

**Comment by Adriani90** [comment]
I am prety sure I have seen this issue also in last NVDA Alpha.

**Comment by Adriani90** [comment]
cc: @jcsteh, @leonardder

**Comment by DrSooom** [comment]
Duplicate of #8730.

**Comment by Adriani90** [comment]
Closing as duplicate. Thank you @DrSooom for your fast reaction.
